convertors.py

```diff
@@ -106,7 +108,7 @@ def ConvertFibitStepsPoint(self, date, data_point):
 		return dict(
 			dataTypeName='com.google.step_count.delta',
 			startTimeNanos=epoch_time_nanos,
-			endTimeNanos=epoch_time_nanos+110,
+			endTimeNanos=epoch_time_nanos + self.NANOS_PER_MINUTE,
```

Comment:
The API performs validation on incoming data points. I think this will limit you to <some reasonable cadence> * 60s as the data point's value.

Reply:
I received errors with 110ns but none with 1m. Fitbit seems to give the data in 1m buckets so this seems ~correct to me. Not sure if it's the start or the end, though.